bitcoin/tests/serde.rs

```diff
@@ -312,6 +312,10 @@ fn serde_regression_psbt() {
         }],
     };
 
+    // Check if the psbt is valid
+    let serialized = psbt.serialize();
+    Psbt::deserialize(&serialized).unwrap();
```

Comment:
This doesn't do validity checks, it just proves we can roundtrip serialize/deserialize the PSBT according to bip-174, right?

Perhaps
```
/// Sanity, check we can roundtrip BIP-174 serialize.
```

Reply:
Yeah! makes much more sense! (I really need to improve my commenting skills :))